common/ai_strategies/03_political_strategies.txt

```diff
@@ -1,4 +1,4 @@
-﻿ai_strategy_hardline_communist = {
+﻿1ai_strategy_hardline_communist = {
```

Comment:
Is this a typo?

Reply:
Err, that's not a vanilla one, probably a typo